templates/layout.html

```diff
@@ -0,0 +1,125 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="shortcut icon" type="image/png" href="{{ base_url }}static/favicon.ico" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nunito:300,400,600,700,800&display=swap">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+        integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css"
+        integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
+
+    <link rel="stylesheet" href="{{ base_url }}static/css/styles.css?v=1.0.2">
+
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+        integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+        integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
+        crossorigin="anonymous"></script>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/js/all.min.js"
+        integrity="sha256-MAgcygDRahs+F/Nk5Vz387whB4kSK9NXlDN3w58LLq0=" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header class="navbar navbar-expand navbar-dark fixed-top flex-column flex-md-row bd-navbar">
+        <a class="navbar-brand mr-0 mr-md-2" href="/" aria-label="PyCon Colombia Icon">
+            <img src="{{ base_url }}static/img/icon.svg" alt="PyCon Colombia Icon" style="width: 50px;">
+        </a>
+        <ul class="navbar-nav flex-row ml-md-auto d-none d-md-flex">
+            <li class="nav-item dropdown">
+                <a class="nav-item nav-link dropdown-toggle mr-md-2" href="#" id="bd-versions" data-toggle="dropdown"
+                    aria-haspopup="true" aria-expanded="false">
+                    v.2021
+                </a>
+                <div class="dropdown-menu dropdown-menu-right" aria-labelledby="bd-versions">
+                    <a class="dropdown-item" href="https://2020.pycon.co">v.2020</a>
+                    <a class="dropdown-item" href="https://2019.pycon.co">v.2019</a>
+                    <a class="dropdown-item" href="https://2018.pycon.co">v.2018</a>
+                    <a class="dropdown-item" href="https://2017.pycon.co">v.2017</a>
+                </div>
+            </li>
+        </ul>
+    </header>
+    <div id="stars2" class="hidden-xs"></div>
```

Comment:
have this div any purpose or we can remove  it?

Reply:
This `div` tag is used to render the dynamic visualization of stars

<img width="1433" alt="Screen Shot 2020-03-29 at 2 36 15 PM" src="https://user-images.githubusercontent.com/14989202/77858706-b5768f80-71ca-11ea-9bd5-f184bfdd8576.png">